README.md

```diff
@@ -7,17 +7,32 @@ This repository contains code to create stores of cloud-optimized GeoTiFFs (COGs
 ### 1. ECMWF SEAS5 Seasonal Forecasts
 
 <details>
+
+
+**[Pipeline docs](src/seas5/README.md)**
+
 These forecasts contain 0.4 degree resolution global data on precipitation rates across 0-6 month lead-times. Historical data from as early as 1981 has been accessed via ECMWF's [Meteorological Archival and Retrieval System](https://www.ecmwf.int/en/forecasts/access-forecasts/access-archive-datasets) (MARS). See this [User Manual](https://www.ecmwf.int/sites/default/files/medialibrary/2017-10/System5_guide.pdf) for more details.
 
-See [this doc](src/seas5/README.md) for details on running the pipeline.
 </details>
 
-### 2. IMERG Global Precipitation Measurement
+### 2. ECMWF ERA5 Reanalysis
```

Comment:
Since this pipeline is being added after IMERG is there some specific reason why this is being bumped up to second here?